packages/agoric-cli/tools/getting-started.js

```diff
@@ -103,6 +103,7 @@ export const gettingStartedWorkflowTest = async (t, options = {}) => {
     if (process.env.AGORIC_INIT_OPTIONS) {
       const opts = JSON.parse(process.env.AGORIC_INIT_OPTIONS);
       initOptions.push(...opts);
+      initOptions['--dapp-template'] = 'dapp-fungible-faucet';
```

Comment:
Pretty sure I commented on this earlier, but this isn't correct. The option needs to be set earlier so it can be overridden by env options.

Reply:
@mhofman I ended up updating the parameter in `registry.sh` directly. CI seems to be passing. What do you think about this approach?